src/WinGetUtil/WinGetUtil.h

```diff
@@ -24,6 +24,50 @@ extern "C"
         ErrorOnVerifiedPublisherFields = 0x2,
     };
 
+    enum WinGetValidateManifestOptionV2
+    {
+        // Below options define what validations to be performed
+ 
+        // No validation, caller will get E_INVALIDARG
+        None = 0,
+        // Only validate against json schema
+        SchemaValidation = 0x1,
+        // Validate against schema and also perform semantic validation
+        SchemaAndSemanticValidation = 0x2,

```

Comment:
Shouldn't this just be `SemanticValidation` and you pass both flags to get both?